### ISSUE TITLE
PythonScript standard output is not shown in Godot Output pannel

### ISSUE THREAD

**Issue by albertas** [touilleMan/godot-python#210]
I have freshly installed Godot_v3.2.2-stable_x11.64 and PythonScript 0.40.0 using AssetLib. Then I have created 2D Scene and renamed it World, Attached Python Script to it with code:

```
from godot import exposed, export
from godot import *


@exposed
class World(Node2D):

	# member variables here, example:
	a = export(int)
	b = export(str, default='foo')

	def _ready(self):
		"""
		Called every time the node is added to the scene.
		Initialization here.
		"""
		print('Ready')
		
	def _unhandled_key_input(self, event,*args,**kwargs):
		print('Got Event')
```
Then started the project using F5, pressed some keyboard keys and no output was printed. I expected to see "Ready" and "Got Event" in the output. However I only got this output:

```
--- Debugging process started ---
Godot Engine v3.2.2.stable.official - https://godotengine.org
OpenGL ES 3.0 Renderer: Mesa DRI Intel(R) Sandybridge Mobile 
 
Pythonscript 0.40.0 (CPython 3.8.3.final.0)
Calling wrapper on <class 'Node2D.Node2D'>
--- Debugging process stopped ---
```
I have repeated the same use case using GDScript, and it worked.

**Comment by matheus2740** [comment]
Did you check the console output? (not inside godot, but the console window)
If you want python output to be captured to godot, go to Project > Project Settings > Python Script > Io Streams Capture

**Comment by albertas** [comment]
I have started the Godot using terminal and standard output of PythonScripts are shown in the terminal, but not in the Godot Output.

When I have Io Streams Capture turned on I get this initial output:
```

--- Debugging process started ---
Godot Engine v3.2.2.stable.official - https://godotengine.org
OpenGL ES 2.0 Renderer: Mesa DRI Intel(R) Sandybridge Mobile 
OpenGL ES 2.0 Batching: ON
 
Pythonscript 0.40.0 (CPython 3.8.3.final.0)
Calling wrapper on <class 'World.World'>
--- Debugging process stopped ---
```
And when its off:
```
--- Debugging process started ---
Godot Engine v3.2.2.stable.official - https://godotengine.org
OpenGL ES 2.0 Renderer: Mesa DRI Intel(R) Sandybridge Mobile 
OpenGL ES 2.0 Batching: ON
 
--- Debugging process stopped ---
```
In both cases standard output is only shown in the Terminal.

**Comment by WilliamTambellini** [comment]
Same bug with me with godot 3.2.2 
and godot-python-assetlib-repo-5dd6d3350ee738986d0d3d3bfab06a76d885071d
and Project settings > Python Script settings > Io streams capture : on
but the python prints dont show in the godot output:
Godot Engine v3.2.2.stable.official - https://godotengine.org
Pythonscript 0.40.0 (CPython 3.8.3.final.0)
PYTHONPATH: ['/home/wtambellini/godot/pythontest/lib', '/home/wtambellini/godot/pythontest/', '/home/wtambellini/godot/pythontest/addons/pythonscript/x11-64/lib/python38.zip', '/home/wtambellini/godot/pythontest/addons/pythonscript/x11-64/lib/python3.8', '/home/wtambellini/godot/pythontest/addons/pythonscript/x11-64/lib/python3.8/lib-dynload', '/home/wtambellini/godot/pythontest/addons/pythonscript/x11-64/lib/python3.8/site-packages', '/home/wtambellini/godot/pythontest/addons/pythonscript/x11-64/lib/python3.8/site-packages/setuptools-46.1.3-py3.8.egg', '/home/wtambellini/godot/pythontest/addons/pythonscript/x11-64/lib/python3.8/site-packages/pip-20.0.2-py3.8.egg']
Loading python script from res://main.py
Calling wrapper on <class 'main.Main'>
Destroying python script Main

**Comment by whogben** [comment]
Same issue here with a fresh install of 0.4, and Io Streams Capture turned on.

The console prints:
```
Pythonscript 0.40.0 (CPython 3.8.3.final.0)
Calling wrapper on <class 'Worlds.test.test'>
```

The code for the test node is:
```
from godot import exposed, export
from godot import *

@exposed
class test(Spatial):

	def _ready(self):
		print("something")
		self.remove_child(self.get_child(0))
```

I am able to confirm that the python is running because I can see the child node disappear in the game window.

Also, on my previous (at least a year old) godot-python setup, with the same Godot editor version as this one, print statements do work.

Anyone got a workaround / can we get this labelled as a bug? At least for me, I have no idea how to debug my godotpython code without print statements.

**Comment by whogben** [comment]
Here's a workaround that can be used to allow development in the meantime:

1. Create a godot script to print from and set it to autoload. (note: autoload as singleton crashes my godot, not sure why, so uncheck that if it happens to you).

_PyPrinter.gd:_
```
extends Node
func p(msg):
	print(msg)
```
2. Access the printer from your python scripts and use it in place of print statements:

_test.py_
```
from godot import exposed, export
from godot import *
@exposed
class test(Spatial):
	def _ready(self):
		print('something')
		self.printer = self.get_node("/root/PyPrinter")
		self.printer.p('something2')
```

Resulting console output:

```
Godot Engine v3.2.1.stable.official - https://godotengine.org
OpenGL ES 3.0 Renderer: AMD Radeon R9 M295X OpenGL Engine
 
Registered camera FaceTime HD Camera (Built-in) with id 1 position 0 at index 0
Pythonscript 0.40.0 (CPython 3.8.3.final.0)
Calling wrapper on <class 'Worlds.test.test'>
CALLING _CALL 'p' on <Node wrapper on 0x7fb5971d5cc0>
something2
```

**Comment by touilleMan** [comment]
this should be fixed by #198
